src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -116,6 +116,7 @@ public void actionPerformed(ActionEvent e) {
             }
         });
         
+        ImageIcon addButtonIcon2 = ImageHelper.createImageIcon("images/add2.jpg");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "addButtonIcon2" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)